README.md

```diff
@@ -1,4 +1,11 @@
-# Alby - Lightning Browser Extension
+<p align="center">
+  <img width="300px" src="/doc/Alby-logo-figure-full.svg">
```

Comment:
Do we still need the png-file?

Reply:
I have added files of both formats，svg、png. I compare them and see that the effect is better

Reply:
Alright, svg should be fine

Reply:
I have deleted the png file